CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Fixed
+
+- mirrord-agent: Handle HTTP upgrade requests when the stealer feature is enabled (with HTTP traffic) PR [#973](https://github.com/metalbear-co/mirrord/pull/973).
```

Comment:
Did you decide to reference the PR instead of the issue because the PR has more information?

Reply:
Yup, as I had created the issue with very little information, as I was going to be working on it asap.